sdk/keyvault/azure-security-keyvault-keys/migration_guide.md

```diff
@@ -0,0 +1,226 @@
+# Guide for migrating to azure-security-keyvault-keys from azure-keyvault
+This guide is intended to assist in the migration to `azure-security-keyvault-keys` from `azure-keyvault`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+Familiarity with the `azure-keyvault` package is assumed. For those new to the Key Vault Key client library for Java, please refer to the [README for name of new package here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-keys/README.md) rather than this guide.
```

Comment:
`README for name of new package here` should have the new package name :)

Reply:
Thanks! I don't know how I missed this :P

Reply:
This is a cool experiment. @sadasant missed the same. I wonder if it has something to do with how we read through things. Now I am waiting to see if the other two language folks miss the exact same thing!

Reply:
I think "For those new to the Key Vault Key client library for Java, please refer to the [README]" reads a lot better than "For those new to the Key Vault Key client library for Java, please refer to the [Key Vault Key client library for Java README]". What do you think @ramya-rao-a?

Reply:
I tend to use multiple words for the link text rather than single words like "here" or "see" because it gives the required context to screen readers when using the "list all links on page" feature. Thus my aversion to using just "README" as the link text. How about this:

> If you are not familiar with the older `azure-keyvault` package, you can skip this migration guide and go over to the [readme for azure-security-keyvault-keys](add link here)